src/index.scss

```diff
@@ -27,41 +27,46 @@
 
 .homePageContent {
   display: flex;
-}
 
-.homePageContent .sideNavBar {
-  flex: 20%;
-  border-style: solid;
-  margin: 20px;
-  text-align: center;
-}
+  .sideNavBar {
+    width: 20%;
+    border-style: solid;
+    margin: 20px;
```

Comment:
avoid the use of `px` at all costs. Everything should be in `rem`

Reply:
fixed